### PR TITLE
[release-2.1] Drop v2v support on release 2.1

### PR DESCRIPTION
**Pull request by tiraboschi**

### CODE REVIEW

**Comment by djzager** [comment]
This must come after #306 since it actually stops the reconcile of the configmap

File: deploy/operator.yaml [hunk 1]
```diff
@@ -29,9 +29,7 @@ spec:
               fieldPath: metadata.name
         - name: WATCH_NAMESPACE
         - name: CONVERSION_CONTAINER
```
Comment (djzager) [inline deploy/operator.yaml]: Can probably remove the environment variable entirely for now. We should probably also drop the `ensure_ims_config` from the hyperconverged controller.

**Comment by mareklibra** [comment]
lgtm in general - to disable the v2v in the UI, it's sufficient to remove the `V2VVmware` CRD which I can see to happen in this PR.

I would need to see this PR executed (which I can't) to be sure that other changes do not break anything else.

**Pull request merged**